assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -22,6 +22,21 @@ angular.module("openshiftConsole")
 
     $scope.projectName = $routeParams.project;
     $scope.sourceURLPattern = SOURCE_URL_PATTERN;
+    $scope.imageName = $routeParams.imageName;
```

Comment:
Adding values to `$scope` is a signal to me that it's used in the view. I usually don't add items there unless I need to. I'd prefer using a local var otherwise.

```
var imageName = $routeParams.imageName;
```

?